hack/overlays/overlay-image.yaml

```diff
@@ -0,0 +1,34 @@
+#@ load("@ytt:overlay", "overlay")
+#! rewrite image and image pull policy
+
+#@ ko_import_path = "ko://github.com/projectcontour/contour/cmd/contour"
+
+#@overlay/match by=overlay.all,expects=17
+#@overlay/match-child-defaults when=1
```

Comment:
I think we should only have `expects="1+"` otherwise I can see this being fairly brittle

or we target specific deployments/daemonsets